messaging/src/test/java/org/axonframework/eventhandling/processors/streaming/token/store/ConfigTokenTest.java

```diff
@@ -33,13 +33,13 @@
  */
 class ConfigTokenTest {
 
-    static Collection<TestSerializer> serializers() {
-        return TestSerializer.all();
+    static Collection<TestConverter> serializers() {
```

Comment:
Can we replace `serializers` for `converters`? 👼 
Also in the other test classes using parameterized tests 🙏